ibis/backends/tests/test_aggregation.py

```diff
@@ -772,6 +772,64 @@ def test_reduction_ops(
         np.testing.assert_array_equal(result, expected)
 
 
+@pytest.mark.parametrize(
+    ("ibis_cond", "pandas_cond"),
+    [
+        param(lambda _: None, lambda _: slice(None), id="no_cond"),
+        param(
+            lambda t: t.string_col.isin(["1", "7"]),
+            lambda t: t.string_col.isin(["1", "7"]),
+            id="cond",
+            marks=[
+                mark.broken(["datafusion"], raises=AssertionError),
```

Comment:
This seems redundant given the `notyet` marker for `datafusion` below on line 820

Reply:
Removed!